query-engine/connectors/sql-query-connector/src/filter_conversion.rs

```diff
@@ -74,17 +75,40 @@ trait AliasedSelect {
     fn aliased_sel(self, alias: Option<Alias>) -> Select<'static>;
 }
 
+trait AliasedColumn {
```

Comment:
I think this should be moved to our `column.rs` file where we have the `as_column` trait. That way we keep or the extensions for columns in one place